sbot/null_feed_test.go

```diff
@@ -239,9 +262,30 @@ func TestNullFetched(t *testing.T) {
 	r.NoError(err)
 
 	mainLog.Log("msg", "check we got all the messages")
-	bobsSeqV, err := alisVersionOfBobsLog.Seq().Value()
-	r.NoError(err)
-	r.EqualValues(1000, bobsSeqV.(margaret.Seq).Seq())
+
+	gotMessage := make(chan struct{})
+	updateSink := luigi.FuncSink(func(ctx context.Context, v interface{}, err error) error {
+		seq, ok := v.(margaret.Seq)
+		if !ok {
+			return fmt.Errorf("unexpected type:%T", v)
+		}
+		s := seq.Seq()
+		if s == 999 {
```

Comment:
Why 999? Maybe reference the total count or use a variable to describe its relation?